includes/class-wc-countries.php

```diff
@@ -350,20 +350,37 @@ public function get_shipping_country_states() {
 	/**
 	 * Gets an array of countries in the EU.
 	 *
-	 * MC (monaco) and IM (isle of man, part of UK) also use VAT.
-	 *
-	 * @param  string $type Type of countries to retrieve. Blank for EU member countries. eu_vat for EU VAT countries.
+	 * @param  string $deprecated Function used to return VAT countries based on this.
 	 * @return string[]
 	 */
-	public function get_european_union_countries( $type = '' ) {
-		$countries = array( 'AT', 'BE', 'BG', 'CY', 'CZ', 'DE', 'DK', 'EE', 'ES', 'FI', 'FR', 'GB', 'GR', 'HU', 'HR', 'IE', 'IT', 'LT', 'LU', 'LV', 'MT', 'NL', 'PL', 'PT', 'RO', 'SE', 'SI', 'SK' );
+	public function get_european_union_countries( $deprecated = '' ) {
+		$countries = array( 'AT', 'BE', 'BG', 'CY', 'CZ', 'DE', 'DK', 'EE', 'ES', 'FI', 'FR', 'GR', 'HU', 'HR', 'IE', 'IT', 'LT', 'LU', 'LV', 'MT', 'NL', 'PL', 'PT', 'RO', 'SE', 'SI', 'SK' );
 
-		if ( 'eu_vat' === $type ) {
-			$countries[] = 'MC';
-			$countries[] = 'IM';
+		if ( ! empty( $deprecated ) ) {
+			wc_deprecated_argument( 'type', '3.9.0', 'Use the WC_Countries::get_vat_countries method instead.' );
+			$countries = get_vat_countries();
```

Comment:
This line throws a fatal error. I believe it should be `$countries = WC_Countries::get_vat_countries();` instead of `$countries = get_vat_countries();`.